src/vs/workbench/api/common/extHostTerminalService.ts

```diff
@@ -823,13 +823,17 @@ export abstract class BaseExtHostTerminalService extends Disposable implements I
 		return index;
 	}
 
-	public getEnvironmentVariableCollection(extension: IExtensionDescription): vscode.EnvironmentVariableCollection {
+	public getEnvironmentVariableCollection(extension: IExtensionDescription): vscode.EnvironmentVariableCollection | { getScopedEnvironmentVariableCollection(scope: vscode.EnvironmentVariableScope | undefined): vscode.EnvironmentVariableCollection } {
```

Comment:
Let's make a new type `IDefaultEnvironmentVariableCollection` for use in the implementation